pylintrc

```diff
@@ -13,7 +13,7 @@
 # limitations under the License.
 
 [MESSAGES CONTROL]
-disable=missing-docstring
+disable=missing-docstring, too-many-locals, redefined-outer-name, anomalous-backslash-in-string, dangerous-default-value, attribute-defined-outside-init, import-error, broad-except, redefined-variable-type, superfluous-parens
```

Comment:
All needed to be disabled? In case of redefined-outer-name, why not change the local name? Or maybe the global name, whatever makes more sense. If it is intentional for these names to be the same (because you want to use the global) then you will need to declare the name as global in the local scope.

Reply:
True, `redefined-outer-name` should definitely be checked. The others I disabled are not that important in my opinion.